bundles/org.openhab.binding.linktap/README.md

```diff
@@ -0,0 +1,240 @@
+when a manual scan is run when adding a LinkTap Gateway. It is however recommended to use **static IP addresses** and add the
+gateways directly using the IP address.
+
+### Devices
+
+Once connected to a LinkTap gateway, the binding will listen for updates of new devices and add them, to the inbox.
+If the gateway cannot publish to openHAB, then the gateway is checked every 2 minutes for new devices, and they are added to the inbox when discovered.
+
+## Binding Configuration
+
+### Bridge Configuration Parameters
+
+| Name                  | Type   | Description                                                                                                                                                    | Recommended Values | Required | Advanced |
+|-----------------------|--------|----------------------------------------------------------------------------------------------------------------------------------------------------------------|--------------------|----------|----------|
+| host                  | String | The hostname / IP address of the gateway device                                                                                                                |                    | Yes      | No       |
+| username              | String | The username if set for the gateway device                                                                                                                     |                    | No       | No       |
+| password              | String | The password if set for the gateway device                                                                                                                     |                    | No       | No       |
+| enableMDNS            | Switch | On connection whether the mDNS responder should be enabled on the gateway device                                                                               | ON                 | No       | Yes      |
+| enforceProtocolLimits | Switch | If ON data outside of the allowed ranges against the protocol will be logged and not sent                                                                      | ON                 | No       | Yes      |
+| enableJSONComms       | Switch | OFF by default for backwards compatibility, if using up to date firmware with no other local network applications set to ON, for more efficient communications | ON                 | No       | Yes      |
+
+
+**NOTE** When enableMDNS is enabled, upon connection to the gateway option "Enable mDNS responder" is switched on 
+
+### LinkTap Device Configuration Parameters
+
+It is recommended to use the Device Id, for locating devices. This can be found in the LinkTap mobile application under 
+Settings->TapLinker / ValveLinker, e.g.
+
+- ValueLinker_1 (D71BC52F004B1200_1-xxxx)
+  - has Device Id "ValveLinker_1"
+  - has Device Name D71BC52F004B1200_1
```

Comment:
The binding will use either Device Id or Device Name. So as device id's may be harder to find / copy correctly, the device name allows user set names for the devices to be used instead to simplify the user experience. (I have seen other protocols where the id's didn't match because of bugs - although I believe this one is much more stable and likely not to suffer this - but hence why I let either be used).

Reply:
In that case why not just stick to the `deviceName`? Is there a benefit to also suppert `deviceId`?

Reply:
I personally prefer deviceId's but I would imagine the deviceName is easier for a lot of people. Device ID's cant change, but are prone to typo's and harder to locate. The device name could change, but typically easier for users. So I believe this gives greater flexibility, and makes it a bit more user friendly.